src/permissions.ts

```diff
@@ -0,0 +1,259 @@
+    ...timelocks.map(account => [ethers.constants.AddressZero, "setInterestRateModel(address)", account]),
+    ...timelocks.map(account => [ethers.constants.AddressZero, "setReduceReservesBlockDelta(uint256)", account]),
+    [ethers.constants.AddressZero, "setProtocolSeizeShare(uint256)", AccountType.NORMAL_TIMELOCK],
+  ];
+};
+
+export const getRewardDistributorPermissionsTimebased = (): string[][] => {
+  return [
+    [ethers.constants.AddressZero, "setRewardTokenSpeeds(address[],uint256[],uint256[])", AccountType.NORMAL_TIMELOCK],
+    [
+      ethers.constants.AddressZero,
+      "setLastRewardingBlockTimestamps(address[],uint256[],uint256[])",
+      AccountType.NORMAL_TIMELOCK,
+    ],
+  ];
+};
+
+export const getRewardDistributorPermissionsBlockbased = (): string[][] => {
+  return [
+    [ethers.constants.AddressZero, "setRewardTokenSpeeds(address[],uint256[],uint256[])", AccountType.NORMAL_TIMELOCK],
+    [ethers.constants.AddressZero, "setLastRewardingBlocks(address[],uint32[],uint32[])", AccountType.NORMAL_TIMELOCK],
+  ];
+};
+
+export const getIRMPermissions = (): string[][] => {
+  return [
+    [ethers.constants.AddressZero, "updateJumpRateModel(uint256,uint256,uint256,uint256)", AccountType.NORMAL_TIMELOCK],
+    [
+      ethers.constants.AddressZero,
+      "updateJumpRateModel(uint256,uint256,uint256,uint256)",
+      AccountType.CRITICAL_TIMELOCK,
+    ],
+    [
+      ethers.constants.AddressZero,
+      "updateJumpRateModel(uint256,uint256,uint256,uint256)",
+      AccountType.FAST_TRACK_TIMELOCK,
+    ],
+  ];
+};
+
+export const getConverterPermissions = (): string[][] => {
+  return [
+    ...accounts.map(account => [ethers.constants.AddressZero, "pauseConversion()", account]),
+    ...accounts.map(account => [ethers.constants.AddressZero, "resumeConversion()", account]),
+    ...timelocks.map(account => [ethers.constants.AddressZero, "setMinAmountToConvert(uint256)", account]),
+    ...timelocks.map(account => [
+      ethers.constants.AddressZero,
+      "setConversionConfig(address,address,ConversionConfig)",
+      account,
+    ]),
+  ];
+};
+
+export const getXVSVaultTreasuryPermissions = (xvsVaultTreasury: string): string[][] => {
+  return [...timelocks.map(account => [xvsVaultTreasury, "fundXVSVault(uint256)", account])];
+};
+
+export const getOmniChainExecutorOwnerPermissions = (omniChainExecutor: string): string[][] => {
+  return [
+    [omniChainExecutor, "setSendVersion(uint16)", AccountType.NORMAL_TIMELOCK],
+    ...accounts.map(account => [omniChainExecutor, "setReceiveVersion(uint16)", account]),
+    [omniChainExecutor, "forceResumeReceive(uint16,bytes)", AccountType.GUARDIAN],
+    ...accounts.map(account => [omniChainExecutor, "setMaxDailyReceiveLimit(uint256)", account]),
+    ...accounts.map(account => [omniChainExecutor, "pause()", account]),
+    [omniChainExecutor, "unpause()", AccountType.GUARDIAN],
+    [omniChainExecutor, "setPrecrime(address)", AccountType.GUARDIAN],
```

Comment:
I think we don't allow the Guardian to execute this function (see https://github.com/VenusProtocol/governance-contracts/blob/develop/helpers/permissions.ts#L391)